modules/govuk/manifests/apps/imminence.pp

```diff
@@ -50,8 +57,10 @@
   $port,
   $enable_procfile_worker = true,
   $sentry_dsn = undef,
-  $mongodb_nodes = undef,
-  $mongodb_name = 'imminence_production',
+  $mongodb_nodes,
+  $mongodb_name,
+  $mongodb_username,
```

Comment:
I had blank defaults for this originally, but what I'm slightly worried about is that if I set a default username and password, potentially the client in staging/production will try to use that to connect to the old cluster, because this file (unlike the others) doesn't seem to be specific to integration. It might be fine, or it might be that the empty string triggers an attempt to login which may or may not work (compare it with, say, a badly written basic auth handler that treats an empty string as different from nil, and then tries to login with a basic auth hash made from an empty username and password). I'm sure the likely thing is that it'll be fine, I just don't understand enough about how this or the clients work and it makes me a bit nervous.